### DISCUSSION TITLE
一些小小的提议

### DISCUSSION THREAD

**Discussion by bin-lin2** [share121/inter-knot#1794] [其他]
无意义的封面：
![screenshot-1723476011725](https://github.com/user-attachments/assets/c72e9d0e-5b99-4aef-abde-ff89c7dca20e)

1.「评论数」（即原「浏览量」）依旧使用原来的图标（那个小眼睛）。毕竟本身就是还原游戏里的网站，感觉还是更还原一些比较好。
而且现在的图标略显突兀，有时还会出现遮挡问题。
![微信截图_20240812223358](https://github.com/user-attachments/assets/a210fef5-fdc8-4bad-acee-7e3a551ec8a1)
只是换回原来的图标，但数目依旧表示「评论数」，各位绳匠心知肚明即可。

2.评论获取失败时，或许可以采用类似下面这种形式，更简单，明了，还能在画风上更加统一。
![微信截图_20240812223543](https://github.com/user-attachments/assets/e4989a72-4d6f-4d86-8c9c-394a83bec2ac)

3.关于【日程】版块，或许可以做成类似备忘录或者便签的东西。在还原界面的同时让绳匠自己添加或修改“任务”。
我想这会给某些人一点帮助，它增加了绳网的实用性，使绳网不再只是一个无聊了打开看看乐子的“网页”，变得更像是可以时刻挂在桌面上的“工具”。

4.添加针对某条评论的举报功能。

5.扩充管理人手，招募一些社区管理员参与到对帖子的管理工作。估摸着有时间有兴趣当这个免费劳动力的应该也不少。（指指自己.jpg）

6.更改举报按钮的位置。
就我个人而言还是更希望up（站长）把网站做的（至少在画面上）更加还原一些。可惜鄙人才刚刚接触到编程的大门，无法出力，只能口头为up提供一个方向。
①对帖子的一键举报
或许可以挪到帖子封面的右上角，且只在鼠标悬停（或被选中）时显示，平时隐藏。
![QQ截图20240812232806](https://github.com/user-attachments/assets/0c370f7e-e391-401f-bc80-b1b5842f8b8b)
（只是举个例子，美观方面具体再调）
②针对某个评论的举报，设置为鼠标悬停在评论中绳匠的名字时，等级的右边旋出举报按钮。

7.如果后面【绳网等级】版块出了，希望可以添加一个切换账号的功能，和浏览器里登的账号分隔开。

我非常喜欢这个绳网，有种突破次元隔阂的感觉。尽管已经出现了诸如打广告啊之类的东西，但名气还没那么大的绳网就像一个独立的小社区，这里的绳匠都有共同话题，github的不方便同时也限制了某种生物群体的侵入。
就像网络中的一片净土。
我也很感激up，从跨域助手这恐怖的更新速度就能看出up同样在这个网站上倾注了心血。如果可以up可以开个爱发电试试，这样大家也能在B站三连外提供支持，至少能帮忙承担一下服务器费用吧。

 防沉，@一下up。
@share121 
希望没有造成不必要的打扰。

**Comment by capacitor1** [comment]
支持

**Comment by Hongxuli2006** [comment]
昨天其实刷怪了，但是被大家齐心协力地给举报掉了
